incubator/cassandra/README.md

```diff
@@ -1,60 +1,84 @@
 # Cassandra
-A Cassandra Chart for Kubernetes
 
-## Install Chart
-To install the Cassandra Chart into your Kubernetes cluster (This Chart requires persistent volume by default, you may need to create a storage class before install chart. To create storage class, see [Persist data](#persist_data) section)
+Cassandra is a distributed NoSQL database management system designed to handle
```

Comment:
I just find it easier to wrap with 80 char linebreaks, same rule most people use for code, if it really bothers people I can totally go the opposite way and let the editor soft break.